aikau/src/main/resources/alfresco/renderers/_XhrActionsMixin.js

```diff
@@ -99,13 +122,28 @@ define(["dojo/_base/declare",
          var nodeRef = lang.getObject("nodeRef", false, this.currentItem);
          if (nodeRef)
          {
+            // depending on how item was loaded site can be coded in different ways
+            var site;
+            if (this.currentItemSiteShortNameKey)
```

Comment:
This configuration option was put in place to allow short-circuit of the "default lookup chain" if a developer is sure enough about how site can be extracted from the current item.